UI/RemoteISOScreen.cpp

```diff
@@ -598,8 +600,8 @@ RemoteISOBrowseScreen::RemoteISOBrowseScreen(const std::vector<std::string> &gam
 void RemoteISOBrowseScreen::CreateViews() {
 	bool vertical = UseVerticalLayout();
 
-	I18NCategory *mm = GetI18NCategory("MainMenu");
 	I18NCategory *di = GetI18NCategory("Dialog");
+	I18NCategory *n = GetI18NCategory("Networking");
```

Comment:
This makes sense :)
But why "rs" for the pointer name ? I think "ri" looks better to represent "RemoteISO".